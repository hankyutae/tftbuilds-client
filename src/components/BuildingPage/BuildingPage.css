.expand-collapse{
  position:absolute;
  top:10px;
  right:10px;
  font-size:100%;
}

.create-builds-page{
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  background-image: linear-gradient( rgba(0, 0, 0, 0.65), rgba(0, 0, 0, 0.65) ), url(/static/media/backsplash.e48a9080.jpg);
  background-repeat: repeat;
  background-position: 50% 0, 50% 0;
  background-size: 150% auto;
}

.create-builds-page h2{
  color: rgb(255, 254, 254);
}

.search-and-build-separator{
  height:30px;
  width:100%;
  background-color:red;
}