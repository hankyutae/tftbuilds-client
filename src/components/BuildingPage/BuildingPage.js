import React from 'react';
import CurrentBuild from '../CurrentBuild/CurrentBuild';
import ChampsSearch from '../ChampsSearch/ChampsSearch';
import './BuildingPage.css';

class BuildingPage extends React.Component {
  render() {
    return (
      <div className="create-builds-page">
        <CurrentBuild/>{/* 
        <div className='search-and-build-separator'>

        </div> */}
        <ChampsSearch locale='build-page'/>
      </div >
    );
  }

}

export default BuildingPage;
