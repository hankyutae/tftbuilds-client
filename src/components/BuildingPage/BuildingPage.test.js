import React from 'react';
import ReactDOM from 'react-dom';
import BuildingPage from './BuildingPage';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
      <BuildingPage/>
  , div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as expected", () => {
  const tree = renderer
    .create( 
      <BuildingPage/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});