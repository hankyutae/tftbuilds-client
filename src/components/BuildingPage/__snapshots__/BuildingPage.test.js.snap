// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="create-builds-page"
>
  <div
    className="current-build-display"
  >
    <h2
      className="current-build-title"
    >
      Your Current Build
    </h2>
  </div>
  <div
    className="champs-search"
  >
    <h2
      className="champs-search-h2"
    >
      Add a Champion
      <div
        className="filter-view-instructions"
      >
        Click on icon to add
      </div>
    </h2>
    <div
      className="view-type-box"
    >
      <div
        className="view-type unselected button-cursor button-hover-lighten"
        onClick={[Function]}
      >
        Card View
      </div>
      <div
        className="view-type selected button-cursor"
        onClick={[Function]}
      >
        Grid View
      </div>
    </div>
  </div>
</div>
`;
